doc/source/reference/commands/select.rst

```diff
@@ -1088,6 +1089,86 @@ more liked entries.
 The ``select`` command outputs records that ``n_likes`` column value
 is equal to or more than ``10`` from ``Entries`` table.
 
+.. _select-n-worksers:
+
+``n_workers``
+"""""""""""""
+
+.. versionadded:: 12.0.5
+
+TODO: translate
+
+.. note::
+
+   This is a experimental feature. Currently, this feature is still not stable.
+
+   This feature requires :doc:`/reference/command/command_version` 3 or later.
+  
+   This feature requires that Apache Arrow is valid in Groonga.
+
+   It depends on package provider whether Apache Arrow is valid or not.
+
+   You can check whether Apache Arrow is valid by checking whether ``apache_arrow`` is ``true`` or not in the result of the :doc:`/reference/commands/status` command.
+
+   If Apache Arrow is invalid, you should build Groonga from the source code with enabling Apache Arrow following the steps in :doc:`/install` or
+   request to enable Apache Arrow to the package provider.
+
+:ref:`select-drilldown` , :ref:`drilldowns <select-advanced-drilldown-related-parameters>` and :ref:`slices <select-slice-related-parameters>` 
+are executed in parallel when this parameter is specified ``-1`` or ``2`` or more.
+
+``drilldown``, ``drilldowns`` and ``slices`` are executed in serial in default.
+In other words, a next process is executed after a current process is finished.
+So, queries tend to take a long time if there are a lot of ``drilldown``, ``drilldowns`` and ``slices``.
+
+``n_workers`` enables to executing independent ``drilldown``, ``drilldowns`` and ``slices`` in parallel.
+The execution time of the total sum of processes can be shourtend by executing them in parallel.
+This parallel execution is done for each ``select`` command.
+
+"independent" means not using ``dorilldowns.table`` to reference the results of other drilldowns or slices.
+
+If they have dependencies, in other words, if using ``dorilldowns.table``, it wait for finish the dependent drilldowns or slices.
+Therefore, the degree of parallelism is reduced if they have dependencies.
+
+Executing in parallel means using multiple CPUs at the same time.
+If executing in parallel even though there is no free CPU resource, it may actually slow down the execution time.
```

Comment:
```suggestion
If executing in parallel without free CPU resource, it may actually slow down the execution time.
```

文章の中に文章を入れると分かりづらくなるので、前置詞に置き換えたほうが分かりやすいです。
もしあえて　”even though"を使っている（リソースがなくてもどうしても使わないといけない状況があるとか）ならば、このままでもいいと思います！

Reply:
without案の方が良さそうなので対応します。